rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc

```diff
@@ -258,51 +285,93 @@ The credential can be deployed in the production phase.
+Use this credential for accessing Amazon Web Services. Fill in the client ID, client secret and region. Using a session token is optional.
+** *OAuth Credentials*
++
+Use this credential for accessing the Microsoft Outlook email application via Microsoft Azure.
++
+*** *OAuth Host*:
++
+The host address used to get the credentials. The default address for Outlook services is https://login.microsoftonline.com/common/oauth2/v2.0/ . This address might change if the customer uses a self hosted service.
+*** *Client ID*:
++
+To authenticate with the OAuth Host, an Azure application has to be defined in the customers space. This Azure application has an id, that is unique in the entire OAuth Host space and allows a user to explicitly grant or revoke access for this Azure application to his account.
++
+For more information how to create this Azure application for Outlook, see https://learn.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app[Quickstart: Register an application with the Microsoft identity platform^]
+*** *Client Secret*
++
+This secret allows RPA Manager to prove to the OAuth Host, that it has received the permission to access user accounts on behalf of the registered Azure application. Without it the authentication is not possible.
+*** *Redirect URI*
++
+This is the URI to which the OAuth Host will redirect after the user has completed the authentication attempt. This redirect URI has to be registered with the Azure application. Only if the entered redirect URI matches one of the registered URI, the OAuth Host will allow the authentication.
+*** *Scopes*
++
+The scopes define, which permission the user grants to the Azure application. By default, we request the following permissions:
++
+**** `offline_access`
++
+(*Required*) Enables access via a refresh token, which you can use to repeatedly log in to the mail services without requiring user interaction each time.
+**** `https://outlook.office.com/IMAP.AccessAsUser.All`
++
+(*Required* when using IMAP) Enables reading and moving emails from the Outlook IMAP server.
+**** `https://outlook.office.com/POP.AccessAsUser.All`
++
+(*Required* when using POP3) Enables reading emails from the Outlook POP3 server.
+**** `https://outlook.office.com/SMTP.Send`
++
+(*Required* when using SMTP) Enables sending emails from the Outlook SMTP server.
+**** `openid email`
++
+Enables RPA Manager to automatically detect the email account used to log in to the Azure application. If this scope is omitted, you must provide an email in RPA Builder.
++
+*** *E-Mail Address*
++
+Email address for accessing Outlook.
+*** *Refresh Token*
++
+Refresh token generated with the authentication properties.
++
+Learn how to xref:rpa-builder::toolbox-variable-handling-credentials-for-oauth.adoc#generating-an-oauth-refresh-token[generate an OAuth refresh token with RPA Builder].
```

Comment:
This link does not appear blue in the preview. It might be OK, but check in the local build just in case it isn't.

Reply:
@Cristian-Venticinque we need to check this somehow preferably before publishing. It's a link to your new RPA Builder documentation not published yet.